### DISCUSSION TITLE
Dialog: Possibility to open dialog without body-scroll-lock

### DISCUSSION THREAD

**Discussion by chessydk** [tailwindlabs/headlessui#647] [Ideas]
It would be a very useful feature to be able to control whether Dialog should do scroll-lock or not. Sometimes its OK to scroll beneath dialog and in these circumstances we avoid the unpleasant yank on windows when scrollbar disappears/reappears.

**Comment by ryan-mahoney** [comment]
@chessydk please see this PR: https://github.com/tailwindlabs/headlessui/pull/850, feedback welcome!
